### ISSUE TITLE
Re implement uid & mail options for user:* commands

### ISSUE THREAD

**Issue by obriat** [drush-ops/drush#4541]
**Is your feature request related to a problem? Please describe.**
As stated in #4020, `--uid` and `--mail options `have been remove form the `user` command in drush 9. It complicates some scripting actions, like block/unblock admin(1) user, specially if you already change its name for security reason.

**Describe the solution you'd like**
Re implement these options as it have been done for `user:login`

**Comment by weitzman** [comment]
Adapt as needed - `drush uli --name=$(drush uinf --mail=foo@example.com --fields=name --format=string)`

**Comment by obriat** [comment]
Sorry I already done the PR
